browser/BUILD.gn

```diff
@@ -66,6 +66,16 @@ group("browser") {
   public_deps = [ "//chrome/browser" ]
 }
 
+if (is_mac) {
+  source_set("mac_features") {
+    sources = [
+      "mac_features.cc",
```

Comment:
I missed the location of these files when I reviewed it. I think this makes more sense in `brave/browser/updater`?

Reply:
Would you like me to open a PR that moves the file?